src/frontend/src/flows/verifiableCredentials/allowCredentials.ts

```diff
@@ -35,6 +41,15 @@ const allowCredentialsTemplate = ({
     onSubmit: (userNumber) => onAllow(userNumber),
   });
 
+  const consentMessage = new Chan<TemplateElement>(html`${consentMessage_}`);
+
+  // Kickstart markdown parsing & sanitizing; once done, replace the consent message
+  void (async () => {
```

Comment:
can this piece of code throw an error and if yes, should we catch it?

Reply:
if anything is thrown there's not much we can do about it so I'm happy to let it escape; wdyt?

Reply:
It there is an error and we do nothing, does the user ends (UX wise) in an unknown state?

If not, all good
If yes, should we inform it something went wrong?
If not, all good
If yes, then we might want to catch the error in some ways.

Reply:
> does the user ends (UX wise) in an unknown state?

nope, they just get the non-rendered markdown!